docs/key-registry.md

```diff
@@ -0,0 +1,93 @@
+# Key Registry
```

Comment:
I'm not convinced this needs its own doc or that key registry is a meaningful concept given how baked in it is to Open Payments :grimacing: 

Open Payments docs already describe:
- https://docs.openpayments.guide/reference/get-payment-pointer-keys
- https://docs.openpayments.guide/reference/post-request `client` field
- https://docs.openpayments.guide/reference/create-incoming-payment, etc `SIGNATURE_INPUT`

and Rafiki admin docs include:
- https://rafiki.dev/docs/reference/mutations#createpaymentpointerkey
- https://rafiki.dev/docs/reference/mutations#revokepaymentpointerkey

Reply:
We'll sort through docs next week. I rather have it here than not have it at all. We may just move it somewhere else next week.